src/typescript/azure-openapi-validator/rules/utilities/rules-helper.ts

```diff
@@ -53,14 +53,21 @@ export async function getResolvedJson(doc: any): Promise<object | undefined> {
   try {
     const parser = new $RefParser()
     const docCopy = JSON.parse(JSON.stringify(doc))
+    /*
+     * remove x-ms-examples
+     */
+    apply(docCopy, `$..['x-ms-examples']`, e => null)
```

Comment:
what does this function do?

Reply:
set x-ms-example to null , as we don't need to check it